### PR TITLE
Feature job pooldata model

### PR DESCRIPTION
**Pull request by salguerod**
Initial PR for feedback only - final PR still outstanding

### CODE REVIEW

File: scripts/sql/mods/mod_1_new_job_pool_table.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+CREATE TABLE `pooled_sts_job` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
```
Comment (eshen1991) [inline scripts/sql/mods/mod_1_new_job_pool_table.sql]: can we change the table name to sts_job_pool

File: src/main/java/com/google/gcs/sdrs/dao/BaseDao.java [hunk 2]
```diff
@@ -31,6 +29,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.gcs.sdrs.dao.model.PooledStsJob;
```
Comment (eshen1991) [inline src/main/java/com/google/gcs/sdrs/dao/BaseDao.java]: google-java-format doesn't seem to be working properly. Please check it is the latest 1.7.0.1

File: scripts/sql/mods/mod_1_new_job_pool_table.sql (hunk 1 above)
Comment (salguerod) [reply scripts/sql/mods/mod_1_new_job_pool_table.sql]: implemented

File: src/main/java/com/google/gcs/sdrs/dao/BaseDao.java (hunk 2 above)
Comment (salguerod) [reply src/main/java/com/google/gcs/sdrs/dao/BaseDao.java]: fixed

**Comment by salguerod** [comment]
@eshen1991 ready for dev

**Pull request merged**